utils/validate_inits.py

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env python3
+
+import os
+import glob
+from setuptools import find_packages, PEP420PackageFinder
+
+
+class NonTrivialPEP420PackageFinder(PEP420PackageFinder):
+    """
+    The PEP420PackageFinder (used by find_namespace_packages) thinks everything
+    looks like a package, even if there are no python files in it. This is a
+    little stricter and only considers directories with python files in it.
+    """
+
+    @staticmethod
+    def _looks_like_package(path):
```

Comment:
This was needed to avoid some false positives, e.g. https://github.com/Unity-Technologies/ml-agents/tree/0.14.1/gym-unity/images